### ISSUE TITLE
Adding folder and filetype icons using new phantom API

### ISSUE THREAD

**Issue by aziz** [aziz/SublimeFileBrowser#125]
Using the inline layout of the phantom we can render a small html file which renders just an image as the file/folder icon.

**Comment by vovkkk** [comment]
We need to find expanded directories when refreshing/renaming, currently it is just `view.find`

Can we access phantoms of a view and content of a phantom? Does not seem so, there is undocumented `view.query_phantom(pid)`, but it returns region.
We could keep expanded paths in settings just like index, mmm… that would change some logic, plus I’m not ready to drop ST2 support — so all in all it is pretty complex feature, on the other side, having proper icons for different files is very tempting.

**Comment by aziz** [comment]
I totally agree. it's just an idea and prospect for future. I still want to use/see phantoms for smaller features on smaller plugins.
